cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
```

Comment:
> my changes for #342 seems to land in start.go after clusterUp call.

right, some changes go there. As well as some changes needed for #177. As I said, we need a bit of a strategy there. Most importantly, we need a better separation between the command and the actual provisioning.